packages/storage/package.json

```diff
@@ -19,6 +19,7 @@
     "README.md"
   ],
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.842.0",
```

Comment:
I think this one should be an optional dependency.
Also, the package version should be locked exactly like `3.842.0` - without the `^`.